exec/cnex/nstring.h

```diff
@@ -42,13 +42,18 @@ TString *string_appendString(TString *s, TString *ns);
 TString *string_appendCString(TString *s, const char *ns);
 TString *string_appendData(TString *s, char *buf, size_t len);
 TString *string_appendChar(TString *s, char c);
+TString *string_appendBuffer(TString *s, const char *buff, size_t len);
```

Comment:
Inconsistent spelling (`buff` vs `buf` two lines above).

Actually, this function seems to be completely redundant as it is identical to `string_appendData`.

Reply:
Removed duplicate function.